app/code/community/Sendsmaily/Sync/etc/system.xml

```diff
@@ -132,6 +132,68 @@
               </depends>
             </sync_period>
 
+            <newsletter_form_section translate="label">
+              <label>Newsletter form</label>
+              <frontend_model>adminhtml/system_config_form_field_heading</frontend_model>
+              <sort_order>40</sort_order>
+              <show_in_default>1</show_in_default>
+              <show_in_website>1</show_in_website>
+              <show_in_store>0</show_in_store>
+            </newsletter_form_section>
+
+            <active_newsletter_form translate="label comment">
+              <label>Enabled</label>
+              <comment><![CDATA[Collect newsletter subscribers from newsletter form. Learn more from our <a href=http://help.smaily.com/en/support/solutions/articles/16000092458-creating-automation-workflows#a4 target=_blank>tutorial how to create welcome emails.</a>]]></comment>
+              <frontend_type>select</frontend_type>
+              <source_model>adminhtml/system_config_source_yesno</source_model>
+              <sort_order>41</sort_order>
+              <show_in_default>1</show_in_default>
+              <show_in_website>1</show_in_website>
+              <show_in_store>0</show_in_store>
+            </active_newsletter_form>
+
+            <active_captcha translate="label comment">
+              <label>Enable CAPTCHA</label>
+              <comment><![CDATA[We strongly advice you to use CAPTCHA to keep bots from polluting subscribers list.<a href=https://www.google.com/recaptcha/admin/create target=_blank>You can create googles reCAPTCHA api keys here.</a>]]></comment>
```

Comment:
Typos:
- "googles" should be "Google";
- "api" should be "API";

Reply:
Done